src/object.d

```diff
@@ -782,6 +782,21 @@ class TypeInfo_Function : TypeInfo
     string deco;
 }
 
+unittest
+{
+    class C
+    {
+       void func() {}
+       void func(int a) {}
+       int func(int a, int b) { return 0; }
+    }
```

Comment:
I think you don't need to define these functions, just declare them - so your coverage won't be affected. Try it out.

```d
class C
{
        void func();
        void func(int a);
        int func(int a, int b);
}
```